### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by C0derLint**
## What's this PR about?
GCI [task](https://codein.withgoogle.com/dashboard/task-instances/4988333500399616/): Design a login page for Fabrik.

Added a login page at `/account/login` and sign up page at `/account/signup` as shown below:
[Old screenshot](https://user-images.githubusercontent.com/44368997/48709664-3cf1dd00-ec2c-11e8-9c59-d863828b6d9d.gif)

### Screenshot
login2

### CODE REVIEW

**Comment by C0derLint** [comment]
Please check @Ram81

**Comment by C0derLint** [comment]
Thank you for the feedback! I will begin implementing the following as soon as possible. Also, as for the form, I've got some pretty good ideas on how to make it look way better ;)

**Comment by C0derLint** [comment]
@Ram81 Update:
- [x] Removed wacky animation effect
- [x] Backend code complete in terms of logging in and logging out.
- [ ] No change in start page as a home page can be made and this can be the start page. I can work on a new homepage once I am done with this task.
- [x] Improved design to match material guidelines. Also added a nav bar and a side bar. Converted the templates so as to be responsive on mobile screens too.
- [ ] I have not converted the login and signup pages to react components yet, I am still researching on this aspect.

File: ide/templates/account/base.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{% extends "base.html" %}
+
+{% load account socialaccount %}
+{% load static %}
+
+{% block head_title %}Fabric - {% block head_title_extra %}Welcome{% endblock %}{% endblock %}
```
Comment (Ram81) [inline ide/templates/account/base.html]: Change `Fabric` to `Fabrik`

File: ide/forms.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+from allauth.account.forms import SignupForm
```
Comment (Ram81) [inline ide/forms.py]: Please re-implement the signup form as react components and isn't there any better way to implement signup feature. why do we a forms.py. Can you do some more research and come up with some better approach. For reference you can see the implementation of EvalAI authentication.

Comment (C0derLint) [reply ide/forms.py]: I see. Let me try reimplementing this in a better way.

**Comment by Ram81** [comment]
@C0derLint please reopen a PR once your work is complete, closing this.

**Pull request closed without merging**